kyc_account/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _kyc_accept_transaction(self, _record, raise_if_not=True):
+        self.ensure_one()
+        if (
+            _record
+            and _record[0]._name == "account.move"
+            and not _record.filtered(lambda l: l.move_type != "entry")
+        ):
+            return True
+        else:
+            return super(ResPartner, self)._kyc_accept_transaction(
+                _record, raise_if_not
+            )
```

Comment:
remove the else, it is redundant, typical python construction:

```
if some condition:
    return X
return Y  # it will only reach this if the condition is not met
```